cmd/util/ledger/migrations/storage_v4.go

```diff
@@ -0,0 +1,174 @@
+package migrations
+
+import (
+	"encoding/hex"
+	"fmt"
+	"runtime"
+
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/runtime/interpreter"
+
+	"github.com/onflow/flow-go/ledger"
+)
+
+func StorageFormatV4Migration(payloads []ledger.Payload) ([]ledger.Payload, error) {
+
+	migratedPayloads := make([]ledger.Payload, 0, len(payloads))
+
+	jobs := make(chan ledger.Payload)
+	results := make(chan struct {
+		key string
+		ledger.Payload
+		error
+	})
+
+	workerCount := runtime.NumCPU()
```

Comment:
is this works on GCP virtual machine env?

Reply:
I guess so, do you have any concerns why it would not?